metricbeat/etc/beat.yml

```diff
@@ -43,4 +43,6 @@ metricbeat.modules:
   period: 10s
   processes: ['.*']
 
+  # if true, exports the CPU usage in ticks, together with the percentage values
```

Comment:
Does not seem to be in master: https://github.com/elastic/beats/blob/master/metricbeat/etc/beat.yml Could you run `git checkout master metricbeat/etc/beat.yml` ?